### PR TITLE
AMS-1368 Logo element renamed

### PR DESCRIPTION
**Pull request by andreychizhov**

### CODE REVIEW

**Comment by tukaef** [comment]
We're also gonna rename `logoCustomImage` header value somehow: `compositeBitmapImageCustomImage` or just `customImage`. @andreychizhov @denisivan0v what do you think?

**Comment by andreychizhov** [comment]
@tukaef i believe we don't really need that renaming. Imho the only header's puprose is to destinguish cases of upload mini image from original logo image. And the name such `compositeBitmapImageCustomImage` is unlikely to be more clear than current.

File: src/AmsMigrator/DTO/Okapi/CustomImage.cs [hunk 1]
```diff
@@ -5,9 +5,9 @@
 
 namespace AmsMigrator.DTO.Okapi
 {
-    public class CustomImage
+    public class SizeSpecificImage
```
Comment (xakep139) [inline src/AmsMigrator/DTO/Okapi/CustomImage.cs]: Rename the file?

Comment (andreychizhov) [reply src/AmsMigrator/DTO/Okapi/CustomImage.cs]: Ok!

**Pull request merged**